src/test/java/ibm/jceplus/junit/openjceplus/multithread/TestRSASignature.java

```diff
@@ -10,38 +10,17 @@
 
 import ibm.jceplus.junit.base.BaseTestRSASignature;
 import ibm.jceplus.junit.openjceplus.Utils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
 
+@TestInstance(Lifecycle.PER_CLASS)
 public class TestRSASignature extends BaseTestRSASignature {
 
-    //--------------------------------------------------------------------------
-    //
-    //
-    static {
+    @BeforeAll
+    public void beforeAll() {
         Utils.loadProviderTestSuite();
+        setProviderName(Utils.TEST_SUITE_PROVIDER_NAME);
+        setKeySize(1024);
     }
-
-    //--------------------------------------------------------------------------
-    //
-    //
-    public TestRSASignature() {
-        super(Utils.TEST_SUITE_PROVIDER_NAME);
-    }
-
-    public void testRSASignature() throws Exception {
-        System.out.println("executing testRSASignature");
-        BaseTestRSASignature bt = new BaseTestRSASignature(providerName);
-        bt.run();
-
-    }
-
-    public static void main(String[] args) {
-        String[] nargs = {
-                ibm.jceplus.junit.openjceplus.multithread.TestRSASignature.class.getName()};
-        junit.textui.TestRunner.main(nargs);
-    }
-    //--------------------------------------------------------------------------
-    //
-    //
-
-}
-
+}
```

Comment:
Add new line.

Reply:
New line added in commit bd27b562b46b98c872982261f4a7cb8bb2145955

Additionally a checkstyle rule was added to enforce this in the future.